src/state_machine/workload.zig

```diff
@@ -503,6 +522,78 @@ pub fn WorkloadType(comptime AccountingStateMachine: type) type {
             return lookup_ids.len;
         }
 
+        fn build_get_account_filter(self: *const Self, body: []tb.AccountFilter) usize {
+            assert(body.len == 1);
+            const account_filter = &body[0];
+            account_filter.* = std.mem.zeroInit(tb.AccountFilter, .{});
```

Comment:
zeroInit is a bit of a footgun, as it's not actually zeroInit, but rather default init. Using `mem.zero` is somewhat more reliable. 

Though, in this case I think the best is to go with explicit init:

```zig
account_filter.* = .{ /* code here */ }
```

to make sure we didn't forget any fields